go/vt/vitessdriver/convert.go

```diff
@@ -109,12 +109,16 @@ func (cv *converter) bindVarsFromNamedValues(args []driver.NamedValue) (map[stri
 	return bindVars, nil
 }
 
-func newConverter(cfg *Configuration) (c *converter, err error) {
-	c = &converter{
-		location: time.UTC,
-	}
+func newConverter(cfg *Configuration) (*converter, error) {
```

Comment:
I didn't feel good about the naked returns here and wrote this to be more idiomatic and straightforward.